tools/spectral/README.md

```diff
@@ -1,11 +1,38 @@
 # Spectral updates
 
-If adding new rules or updating .spectral.yaml overall, the validations will instantly get updated across mongodb/openapi repository.
+If adding new rules or updating `.spectral.yaml` overall, the validations will instantly get updated across the `mongodb/openapi` repository.
 
-To propagate the changes in MMS, engineers must open a PR and update the pinned commit sha in mms.
+To propagate the changes in MMS, engineers must open an MMS PR and update the pinned commit SHA for the imported spectral file from `mongodb/openapi`.
+
+## Scenarios
+
+There are two scenarios depending on the changes you are making to `.spectral.yaml` and whether it affects the currently published OAS or not.
+
+### Scenario 1: The current OAS doesn't violate the new linting
 
 Please perform the following steps:
-1. [ ] Open a PR with the spectral changes and validate that the Spectral lint checks work.
-2. [ ] Review and merge the PR.
-3. [ ] Open a PR in mms, updating the commit sha of the spectral file imported.
-4. [ ] Validate all tests pass.
\ No newline at end of file
+
+1. Open a `mongodb/openapi` PR with the changes to `tools/spectral/.spectral.yaml`
+2. Validate that the new Spectral lint checks pass
+3. Review and merge the PR
+4. Open a PR in MMS, updating the commit SHA of the imported spectral file
```

Comment:
FYI: This repository is public. For public repositories, we keep internal documentation in wikis. 
This way you can put links and all information without being restricted in any way.

Reply:
Yes, though I think the current content is alright to keep here (also suggested by @blva to update this README). @andreaangiolillo Are you good with keeping the contents here, or would you prefer an internal wiki linked through a golink instead?

Reply:
Went with moving the contents to the wiki and keeping the README lighter for public contributions, RFAL!